pkg/blockatlas/marshal.go

```diff
@@ -110,12 +109,8 @@ func (a *Amount) MarshalJSON() ([]byte, error) {
 }
 
 // Sort sorts the response by date, descending
-func (r *TxPage) Sort() {
-	sort.Slice(*r, func(i, j int) bool {
-		ti := cast.ToUint64((*r)[i].Date)
-		tj := cast.ToUint64((*r)[j].Date)
-		return ti >= tj
-	})
+func (txs *TxPage) Sort() {
+	sort.Sort(txs)
```

Comment:
do we have test for this?  good either way

Reply:
I can do that